core/config/project_settings.cpp

```diff
@@ -145,7 +145,8 @@ String ProjectSettings::localize_path(const String &p_path) const {
 		return p_path.simplify_path();
 	}
 
-	DirAccessRef dir = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	Ref<DirAccess> dir = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
+	ERR_FAIL_COND_V(dir.is_null(), "");
```

Comment:
Maybe it could return `path` defined just below (so move this after the definition of `path`).

Reply:
This check probably is not require at all, `create(DirAccess::ACCESS_FILESYSTEM)` should not fail.